plugins/slicer/MONAILabelReviewer/MONAILabelReviewerLib/MonaiServerREST.py

```
@@ -57,6 +57,28 @@ def getDicomDownloadUri(self, image_id: str) -> str:
         logging.info(f"{self.getCurrentTime()}: REST: request dicom image '{download_uri}'")
         return download_uri
 
+    def requestImage(self, image_id: str):
+
+        try:
+            response = requests.get(download_uri, timeout=30)
```

Ruff (0.16.0)

[error] Undefined name `download_uri`

(F821)